tapestry.php

```diff
@@ -21,6 +22,7 @@
  */
 require_once dirname(__FILE__).'/endpoints.php';
 require_once dirname(__FILE__).'/tapestry-settings.php';
+require_once dirname(__FILE__).'/tapestry-plugin-updates.php';
```

Comment:
Could we please drop the `tapestry-` prefix in both these files? It's clear that it's regarding tapestry since we are in the tapestry plugin directory.